controllers/opportunity/index.js

```diff
@@ -64,6 +64,49 @@ class opportunityController {
       });
     }
   }
+
+  //updateOpportunities
+async updateOpportunity(req,res){
+
+  let opportunityId=req.params.opportunityId;
+  let queryObject={opportunityId:opportunityId}
+  const {
+    opportunityTitle,
+    opportunityType,
+    opportunityOrganisation,
+    opportunityLocation,
+    opportunityDescription,
```

Comment:
What if not all the fields are provided in the req.body? 
A user seldom tries to update the whole object, 
if any of these is missing in req.body, it will be set to `null` in original document after update. 

Basically, your code should update only the fields which you have received in the req.body 
PS: This change in code will bring changes in tests, documentations also  

CC: @abdus

Reply:
Ook @Manvityagi  I will try to update it with all mentioned changes.

Reply:
Hi @Sukriti-sood, Any update on the PR?

Reply:
Sorry  @Manvityagi  , I will update in a day or two.

Reply:
![Screenshot from 2021-04-26 14-50-55](https://user-images.githubusercontent.com/55010599/116067596-e9aad100-a6a6-11eb-9a41-c801737e7af4.png)